tests/bud.bats

```diff
@@ -3864,6 +3864,99 @@ _EOF
   run_buildah rmi myalpine
 }
 
+@test "build test pushing and pulling from remote cache sources" {
+  _prefetch alpine
+  mytmpdir=${TEST_SCRATCH_DIR}/my-dir
+  mkdir -p ${mytmpdir}
+  cat > $mytmpdir/Containerfile << _EOF
+FROM alpine
+RUN echo hello
+RUN echo world
+RUN touch hello
+
+FROM alpine
+RUN echo hello
+COPY --from=0 hello .
```

Comment:
Is this supposed to hit the cache? I don’t see specific checks for this build step below (and it would be a bit useful to have tests specifically exercise the ADD/COPY path).

Reply:
(This testing coverage concern is non-blocking from my POV.)

Reply:
Yes updated tests to cover `COPY` and also added case for `ADD`